src/view/com/util/Toast.tsx

```diff
@@ -7,13 +7,15 @@ import {colors} from 'lib/styles'
 import {useTheme} from 'lib/ThemeContext'
 import {usePalette} from 'lib/hooks/usePalette'
 import {useAnimatedValue} from 'lib/hooks/useAnimatedValue'
+import {IS_TEST} from '#/env'
 
 const TIMEOUT = 4e3
 
 export function show(
   message: string,
   _icon: FontAwesomeProps['icon'] = 'check',
 ) {
+  if (IS_TEST) return
```

Comment:
Toasts also conceal UI and fail tests. We should really just test Toasts separately.